website/content/userguide/permanent-data.markdown

```diff
@@ -0,0 +1,97 @@
+---
+title: "Access permanent data"
+menuTitle: "Access permanent data"
+weight: 49
+---
+
+## Create the filesystem
+
+During the installation of gokrazy on your SD card, 4 partition will be created.
+The first 3 partitions will be used for 1x boot partitions and 2x root partitions.
+The 2x root partitions allows gokrazy to update your host installation on the fly via the HTTP endpoint.
+
+The forth partition can be used to store permanent data.
+Before this free space can be used as persistence, a new filesystem has to be created.
+You can use a variety of tools to do that.
+If you run a Linux distribution you can use the command line tool `mkfs.ext4` or the `gparted` utility if you prefer a GUI.
+In booth cases you'll need to run the utility with root privileges.
+
+Before creating the filesystem, make sure to choose the correct device.
+
+List all available devices:
+
+```bash
+ll /dev/sd*
+
+brw-rw---- 1 root disk 8,  0 Dec 30 19:57 /dev/sda
+brw-rw---- 1 root disk 8, 16 Dec 30 19:57 /dev/sdb
+brw-rw---- 1 root disk 8, 17 Dec 30 19:58 /dev/sdb1
+brw-rw---- 1 root disk 8, 32 Dec 30 20:47 /dev/sdc
+brw-rw---- 1 root disk 8, 33 Dec 30 20:47 /dev/sdc1
+brw-rw---- 1 root disk 8, 34 Dec 30 20:47 /dev/sdc2
+brw-rw---- 1 root disk 8, 35 Dec 30 20:47 /dev/sdc3
+brw-rw---- 1 root disk 8, 36 Dec 30 20:47 /dev/sdc4
+```
+As you can see there is only one device with 4 available partitions.
+`/dev/sdc4` is the target partition for the new filesystem.
+gokrazy support two kinds of filesystems for permanent data: `ext4` and `vfat`.
+
+Create the filesystem with the `mkfs.ext4` utility:
+
+```bash
+sudo mkfs.ext4 /dev/sdc
+```
+After the filesystem was created successfully, gocrazy is able to mount it.
+During the startup, gokrazy will try to mount the forth partition.
+The target mount directory is `/perm`.
+Under this mount point directory data can be stored permanent and will present after gokrazy and application updates.
+
+## Permanent data in action
+
+Accessing permanent data is straight forward. 
+The followed example will demonstrate how to access a config file.
+The config file has to be placed on the filesystem manually.
+Make sure to set proper permissions:
```

Comment:
The following commands are to be done interactively via breakglass, but the text doesn’t mention what breakglass is. Can you add a sentence?